css/main.css

```diff
@@ -576,7 +650,642 @@ footer {
+    }
+}
+
+@media (max-width: 768px) {
+    * {
+        font-size: 14px;
+    }
+
+    h1 {
+        font-size: 40px;
+    }
+
+    h2 {
+        font-size: 32px;
+        line-height: 46px;
+    }
+
+    h3 {
+        font-size: 16px;
+    }
+
+    .email-screen__lower-text {
+        font-size: 11px;
+    }
+
+    .welcome-screen__slogan-block {
+        font-size: 40px;
+        line-height: 48px;
+    }
+
+    .examples-screen {
+        padding: 80px 20px;
+    }
+
+    .examples-screen__advantages {
+        flex-direction: column;
+    }
+
+    .high-dev-standards__upper-block {
+        flex-direction: column;
+        padding-bottom: 40px;
+    }
+
+    .hds-upper-block__text-block {
+        margin: 0;
+        padding: 0 30px;
+    }
+
+    .hds-upper-text-block__title {
+        text-align: center;
+    }
+
+    .hds-upper-block__picture-block {
+        text-align: center;
+        margin-top: 30px;
+    }
+
+    .high-dev-standards__lower-block {
+        flex-wrap: wrap;
+        margin-top: 40px;
+    }
+
+    .hds-lower-block__project-block {
+        flex-basis: 29%;
+        margin-top: 30px;
+    }
+
+    .examples-screen__link_margin-top {
+        margin-top: 0;
+    }
+
+    .jobs-screen__text-block {
+        padding-bottom: 212px;
+    }
+
+    .jobs-wrapper {
+        margin-top: -142px;
+    }
+}
+
+@media (max-width: 680px) {
+    .f__links-list {
+        top: 0;
+    }
+    .f__block {
+        flex-basis: 44%;
+    }
+
+    .footer-wrapper {
+        height: 600px
+    }
+
+    .f__block-1 {
+        margin-left: 13px;
+    }
+
+    .f__block-3 {
+        margin-left: 13px;
+        margin-top: 40px;
+    }
+
+    .f__block-4 {
+        margin-top: 40px;
+        margin-right: 13px;
+    }
+}
+
+@media (max-width: 640px) {
+    .jobs-screen__text-block {
+        flex-direction: column;
+        text-align: center;
+    }
+
+    .jobs-screen__title {
+        margin-right: 0;
+    }
+
+    .jobs-screen__link {
+        margin-top: 50px;
+        margin-left: 0;
+    }
+
+    .jobs-list {
+        flex-direction: column;
+    }
+
+    .job:nth-of-type(2) {
+        margin-left: 0;
+        margin-right: 0;
+    }
+
+    .our-projects__examples {
+        flex-wrap: wrap;
+    }
+
+    .our-projects__example {
+        flex: 0;
+        flex-basis: 44%;
+    }
+
+    .our-projects__example-1 {
+        margin-left: 15px;
+    }
+
+    .our-projects__example-4 {
+        margin-right: 15px;
+    }
+
+    .our-projects__example-3, .our-projects__example-4 {
+        margin-top: 15px;
+    }
+
+    .video-screen__text-block {
+        padding-bottom: 310px;
+    }
+
+    .video-wrapper {
+        margin-top: -240px;
+    }
+
+    .our-projects__text {
+        padding: 50px 90px 70px;
+    }
+}
+
+@media (max-width: 539px) {
+    .our-projects__examples {
+        flex-direction: column;
+    }
+
+    .our-projects__example-2 {
+        margin-top: 15px;
+    }
+}
+
+@media (max-width: 480px) {
+    * {
```

Comment:
It's hard rule, please, list tags you need them have `14px` font weight instead of this .